src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/SelectExpandNode.cs

```diff
@@ -155,7 +155,19 @@ public SelectExpandNode(SelectExpandClause selectExpandClause, IEdmStructuredTyp
         /// <summary>
         /// Gets the list of EDM navigation properties to be expanded in the response.
         /// </summary>
-        public IDictionary<IEdmNavigationProperty, SelectExpandClause> ExpandedNavigationProperties { get; private set; }
+        public IDictionary<IEdmNavigationProperty, SelectExpandClause> ExpandedNavigationProperties
```

Comment:
>ExpandedNavigationProperties [](start = 71, length = 28)

can we make this dictionary read-only?  Adding to it is not really going to do anything, right?